paymentsheet/src/test/java/com/stripe/android/customersheet/CustomerSheetViewModelTest.kt

```diff
@@ -426,6 +426,57 @@ class CustomerSheetViewModelTest {
         }
     }
 
+    @Test
+    fun `When CustomerViewAction#OnPrimaryButtonPressed for saved payment method, selected result is emitted`() = runTest {
```

Comment:
Nit: We should name our tests independent of the implementation.

```suggestion
    fun `When primary button is pressed for saved payment method, selected payment method is emitted`() = runTest {
```